Assets/Plugins/Source/Editor/EditorWindows/EOSSettingsWindow.cs

```diff
@@ -256,13 +256,17 @@ private void Save()
             _productConfigEditor.Save();
 
             // reload the product config editor
-            _productConfigEditor.Load();
 
             // Save each of the platform config editors.
             foreach (IConfigEditor editor in _platformConfigEditors)
             {
                 editor.Save();
             }
+#if UNITY_EDITOR
+            AssetDatabase.SaveAssets();
+            AssetDatabase.Refresh();
+#endif
```

Comment:
Minor: Unnecessary #if check, this is all Editor code
```suggestion
            AssetDatabase.SaveAssets();
            AssetDatabase.Refresh();
```

Reply:
Removed #if check